GameServer/Logic/Content/ItemManager.cs

```diff
@@ -183,12 +183,10 @@ public bool GetIsTrinket()
 
     public class ItemRecipe
     {
-        private Game _game;
         private ItemType _owner;
         private ItemType[] _items;
         private int _totalPrice;
-
-        public List<ItemType> Items { get { if (_items == null) FindRecipeItems(); return _items.ToList(); } }
```

Comment:
This thing and relevant changes seem still undone, so I won't mind it being like this for now, but I'd like to see the caching / C# type properties being used when you get around refactoring this.